### PR TITLE
SonarQube fixes

### PR DESCRIPTION
**Pull request by gverm**
http://www.playonlinux.org:9000/issues#issues=AVw-tQUD9U_f07q6H1DW
http://www.playonlinux.org:9000/issues#issues=AVw-tQAL9U_f07q6H0_W
http://www.playonlinux.org:9000/issues#issues=AVw-tQHA9U_f07q6H1Aw
http://www.playonlinux.org:9000/issues#issues=AVw-tQQc9U_f07q6H1Cp
http://www.playonlinux.org:9000/issues#issues=AVw-tQPW9U_f07q6H1Cb

### CODE REVIEW

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ protected void drawStepContent() {
 
         textField = new TextField();
         textField.textProperty().addListener((observable, oldValue, newValue) -> {
-            if (newValue.trim().equals("")) {
+            if ("".equals(newValue.trim())) {
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: I think `newValue.trim().isEmpty()` would be even better here

**Comment by madoar** [comment]
ok to test

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java (hunk 1 above)
Comment (qparis) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: StringUtils.isBlank is even better

Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: You're right, use `StringUtils.isBlank`

Comment (gverm) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: Done.

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java [hunk 2]
```diff
@@ -42,7 +43,7 @@ protected void drawStepContent() {
 
         textField = new TextField();
         textField.textProperty().addListener((observable, oldValue, newValue) -> {
-            if (newValue.trim().equals("")) {
+            if (StringUtils.isBlank(newValue.trim())) {
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: As far as I understand `StringUtils.isBlank` a trim isn't needed anymore. 
Can you please take a look if this is the case? If this is the case please remove the `trim()` call.

Comment (gverm) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java]: @madoar You are right, sorry about that.

**Comment by gverm** [comment]
@madoar @qparis I forgot to include one fix, I will push it now, sorry about that.

**Pull request merged**